test/unit/ndr_error/log_test.rb

```diff
@@ -398,5 +385,14 @@ def setup
       error = simulate_raise(Exception, 'msg', %w( bob/local-app-copy/foo bob/gems/bar ))
       assert_equal %w( bob/local-app-copy/foo ), error.application_trace
     end
+
+    test 'should raise when trying to generate SQL from non-existant user_column' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.